### PR TITLE
add meta properties

### PR DESCRIPTION
**Pull request by cp-mansi-d**
**I have minor change on header.js, becoz i was see error on header and get solution this error ,so i have sloved this error.**

### CODE REVIEW

File: components/partials/header.js [hunk 1]
```diff
@@ -26,10 +26,10 @@ export default function Header() {
...
     <>
       <header
-        className={`sticky w-full top-0 z-10 bg-white ${
+        className={`w-full top-0 z-10 bg-white ${
           showHeader
-            ? `visible ${lastScrollPos > 0 ? "drop-shadow-lg" : ""}`
-            : "invisible"
+            ? `sticky ${lastScrollPos > 0 ? "drop-shadow-lg" : ""}`
+            : "relative"
         }`}
```
Comment (cp-mansi-d) [inline components/partials/header.js:33]: i have change this becoz i was see error on header and i was get solution this error ,so i have slove the error.

File: pages/post/[slug].js [hunk 2]
```diff
@@ -33,6 +33,8 @@ export default function Post({ post, status }) {
   if (post) {
     post = post.attributes;
     var published_on = post.published_on.replace(",", "");
+    var published_time = new Date(post.publishedAt).toLocaleTimeString();
+    var authorName = post.author_id.data.attributes.username;
```
Comment (cp-sumi-k) [inline pages/post/[slug].js:37]: Let's assign the author directly to the post in `setPostFields` method. So its easy for us to get it directly. Do same for image also

File: pages/seo.js [hunk 3]
```diff
@@ -1,15 +1,120 @@
...
+            property="article:published_time"
+            content={publishedTime}
+          />
+          <meta data-rh="true" property="title" content={title} />
+          <meta data-rh="true" property="og:title" content={title} />
+          <meta data-rh="true" property="twitter:title" content={title} />
+          <meta
+            data-rh="true"
+            name="description"
+            content={description.substring(0, 200) + "..."}
```
Comment (cp-sumi-k) [inline pages/seo.js:45]: I think it will not be effective. Let's add and use the `meta_description` field from Strapi

File: pages/seo.js [hunk 4]
```diff
@@ -1,15 +1,120 @@
...
+  url,
+  image_url,
+  publishedAt,
+  publishedTime,
+  readingTime,
+  article,
+}) => {
   return (
     <Head>
-      <meta charset="UTF-8" />
```
Comment (cp-sumi-k) [inline pages/seo.js:7]: Charset, robots and viewport are mandatory headers for any website. Let's not remove it. Also let's add meta properties for home page also. Refer ticket for content

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

File: pages/seo.js [hunk 5]
```diff
@@ -1,15 +1,115 @@
...
+            data-rh="true"
+            property="og:description"
+            content={description.substring(0, 200) + "..."}
+          />
+          <meta
+            data-rh="true"
+            property="twitter:description"
+            content={description.substring(0, 200) + "..."}
+          />
+          <meta data-rh="true" property="og:url" content={url} />
```
Comment (cp-sumi-k) [inline pages/seo.js:52]: Let's remove `data-rh` from everywhere

File: utils.js [hunk 6]
```diff
@@ -32,6 +33,16 @@ function setPostFields(post) {
   post.attributes.published_on = date;
   post.attributes.readingTime = getReadingTime(post.attributes.content);
   post.attributes.image_url = post.attributes.image.data.attributes.url;
+  post.attributes.authorName =
```
Comment (cp-sumi-k) [inline utils.js:36]: Let's make common author like,
`const author = post.attributes.author_id.data.attributes` then use it like, 
`post.attributes.authorName = author.username`

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

**Review by cp-sumi-k — APPROVED** [review]

**Pull request merged**